core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -100,77 +49,14 @@ public TableScan appendsAfter(long fromSnapshotId) {
-    Preconditions.checkArgument(
-        tableOps().current().snapshot(scanSnapshotId) != null,
-        "Cannot find snapshot with ID %s",
-        scanSnapshotId);
-    return newRefinedScan(
-        tableOps(), table(), tableSchema(), context().useSnapshotId(scanSnapshotId));
-  }
-
-  @Override
-  public TableScan useRef(String name) {
-    Preconditions.checkArgument(
-        snapshotId() == null, "Cannot override ref, already set snapshot id=%s", snapshotId());
-    Snapshot snapshot = table().snapshot(name);
-    Preconditions.checkArgument(snapshot != null, "Cannot find ref %s", name);
-    return newRefinedScan(
-        tableOps(), table(), tableSchema(), context().useSnapshotId(snapshot.snapshotId()));
-  }
-
-  @Override
-  public TableScan asOfTime(long timestampMillis) {
-    Preconditions.checkArgument(
-        snapshotId() == null, "Cannot override snapshot, already set snapshot id=%s", snapshotId());
-
-    return useSnapshot(SnapshotUtil.snapshotIdAsOfTime(table(), timestampMillis));
-  }
-
-  @Override
-  public CloseableIterable<FileScanTask> planFiles() {
-    Snapshot snapshot = snapshot();
-    if (snapshot != null) {
-      LOG.info(
-          "Scanning table {} snapshot {} created at {} with filter {}",
-          table(),
-          snapshot.snapshotId(),
-          DateTimeUtil.formatTimestampMillis(snapshot.timestampMillis()),
-          ExpressionUtil.toSanitizedString(filter()));
-
-      Listeners.notifyAll(new ScanEvent(table().name(), snapshot.snapshotId(), filter(), schema()));
-      List<Integer> projectedFieldIds = Lists.newArrayList(TypeUtil.getProjectedIds(schema()));
-      List<String> projectedFieldNames =
-          projectedFieldIds.stream().map(schema()::findColumnName).collect(Collectors.toList());
-
-      Timer.Timed planningDuration = scanMetrics().totalPlanningDuration().start();
-
-      return CloseableIterable.whenComplete(
-          doPlanFiles(),
-          () -> {
-            planningDuration.stop();
-            Map<String, String> metadata = Maps.newHashMap(context().options());
-            metadata.putAll(EnvironmentContext.get());
-            ScanReport scanReport =
-                ImmutableScanReport.builder()
-                    .schemaId(schema().schemaId())
-                    .projectedFieldIds(projectedFieldIds)
-                    .projectedFieldNames(projectedFieldNames)
-                    .tableName(table().name())
-                    .snapshotId(snapshot.snapshotId())
-                    .filter(ExpressionUtil.sanitize(filter()))
-                    .scanMetrics(ScanMetricsResult.fromScanMetrics(scanMetrics()))
-                    .metadata(metadata)
-                    .build();
-            context().metricsReporter().report(scanReport);
-          });
-    } else {
-      LOG.info("Scanning empty table {}", table());
-      return CloseableIterable.empty();
-    }
+  /**
```

Comment:
I'm not sure why this is marked as new code, looks like it's the same as the existing deprecated code

Reply:
I'll assume google is just doing weird diffing?

Reply:
Actually looks like its removed recently , removed this method in the rebase.